app/src/main/res/values/strings.xml

```diff
@@ -366,22 +366,22 @@
     <string name="keyboard_theme_title">Keyboard theme</string>
     <string name="keyboard_keys_category">Keys</string>
     <string name="keyboard_auto_go_action_title">Auto key action</string>
-    <string name="keyboard_auto_go_action_summary">Action of the "Go" key performed automatically after pressing a "Field" key</string>
+    <string name="keyboard_auto_go_action_summary">\"Go\" key action after pressing a \"Field\" key</string>
     <string name="keyboard_key_vibrate_title">Vibratory keypresses</string>
     <string name="keyboard_key_sound_title">Audible keypresses</string>
     <string name="allow_no_password_title">Allow no master key</string>
-    <string name="allow_no_password_summary">Enable the \"Open\" button if no credentials are selected</string>
+    <string name="allow_no_password_summary">Shows \"Open\" button if no credentials are selected</string>
     <string name="delete_entered_password_title">Delete password</string>
     <string name="delete_entered_password_summary">Deletes the password entered after a connection attempt</string>
     <string name="enable_read_only_title">Write-protected</string>
     <string name="enable_read_only_summary">Open your database read-only by default</string>
     <string name="enable_auto_save_database_title">Autosave database</string>
-    <string name="enable_auto_save_database_summary">Automatically save the database after an important action (only in \"Modifiable\" mode)</string>
-    <string name="enable_education_screens_title">Educational screens</string>
-    <string name="enable_education_screens_summary">Highlight the elements to learn how the app works</string>
-    <string name="reset_education_screens_title">Reset educational screens</string>
-    <string name="reset_education_screens_summary">Show all educational items again</string>
-    <string name="reset_education_screens_text">Educational screens reset</string>
+    <string name="enable_auto_save_database_summary">Save the database after every important action (when in \"Modifiable\" mode)</string>
+    <string name="enable_education_screens_title">Educational hints</string>
+    <string name="enable_education_screens_summary">Highlight elements to learn how the app works</string>
+    <string name="reset_education_screens_title">Reset educational hints</string>
+    <string name="reset_education_screens_summary">Show clickable educational info again</string>
+    <string name="reset_education_screens_text">Educational hints reset</string>
```

Comment:
What tense is this in? What happens. Needs a comment.

Reply:
Setting to reset education screens, display a toast

Reply:
And it can't be "Reset all educational hints"?

Reply:
It can, it's good for me